code/index.html

```diff
@@ -4,27 +4,95 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    <!-- 1. HERO IMAGE -->
+    <header class="hero-img">
+      <picture>
+        <source srcset=".\images\hero_img_mobile3.jpg" media="(max-width: 776px)"/>
+        <source srcset=".\images\hero_img_tablet3.jpg" media="(max-width: 1190px)"/>
+        <img src="./images/hero_img_desktop3.jpg" alt="Hero Image Drums">
+      </picture>
+      <div class="hero-text">
+        <h1>GAME OF DRUMS</h1>
+        <h2>Learn it the easy way...</h2>
+      </div>        
+    </header>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <!-- 2. SIGNUP FORM -->
+    <div class="container">
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <form class="form-wrapper" action="https://httpbin.org/anything" method="POST" > <!--autocomplete when writing the form-->
+      <div class="contact-form">
+        <div class="input-fields">
+          <ul>
+            <li>
+              <h2>Contact Us:</h2>
+            </li>
+            <li>
```

Comment:
It gets a little bit cluttered with the `ul` and `li` elements. See if you can style the labels and inputs instead